src/provider/camunda-platform/properties/ListenerProps.js

```diff
@@ -414,6 +414,7 @@ function Field(props) {
   return (
     <CollapsibleEntry
       id={ fieldId }
+      element={ element }
```

Comment:
So indeed we could have detected this if there'd been some mechanism for required props... 😓

Reply:
Well in any case element isn't technically required here, if your implementation of adders and setters doesn't make use of it. CollapsibleEntry is properties-panel core, from my understanding it should be completely unaware of bpmn-js concerns.